app/models/admin_set.rb

```diff
@@ -1,5 +1,7 @@
 # There is an interplay between an AdminSet and a PermissionTemplate.
 # @see Hyrax::AdminSetBehavior for further discussion
 class AdminSet < ActiveFedora::Base
+  DEFAULT_ID = 'admin_set/default'.freeze
```

Comment:
While I can understand (possibly) overriding this constant for applications built on Hydra, I'm not sure we want to *encourage* it. This is supposed to be a constant. Allowing it to be easily changed (via configuration) seems like a different use case (i.e. configuring / changing which admin set you want to be the "default" one). That's not part of this ticket.

So (when I get a chance, later today), I am going to refactor/update this PR based on @mjgiarlo's comments (and to make it a bit more DRY than it currently is).  If we want to provide a new use case to configure your default set, then I'd recommend creating a new ticket for that.